v2/mfa/pre-built-ui/protecting-routes.mdx

```diff
@@ -77,7 +78,7 @@ function App() {
             <div className="App">
                 <div className="fill">
                     <Routes>
-                        {getSuperTokensRoutesForReactRouterDom(reactRouterDom)}
+                        {PasswordlessPreBuiltUI.getReactRouterDomRoutes(reactRouterDom)}
```

Comment:
im not sure if this is correct even.. cause this uses emailpassword as first factor as well?

Reply:
This part only shows adding the new routes, but I agree, it should also show the others.